app/src/lib/git/status.ts

```diff
@@ -207,7 +210,7 @@ export async function getStatus(
       mergeHeadFound,
       rebaseInternalState
     )
-  } else {
+  } else if (entries.length > 0) {
```

Comment:
added this little optimization so we don't run these new git commands as often

Reply:
Can this comment be included in the source? It feels important enough to keep in mind the next time we're in here making changes...

Reply:
done in b055353 :)